### DISCUSSION TITLE
HC_MISC/stego/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#63] [Show and tell]
# HC_MISC/stego/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://ctf.tj.cn/HC_MISC/stego/

**Comment by Transmigration-zhou** [comment]
https://georgeom.net/StegOnline/  一个 web 版能解决部分图片隐写的工具

**Comment by asdcvxsda** [comment]
怎么下string wbm lsb... 的命令

**Reply by n-WN** [reply]
It's recommended to directly use Linux, then use the Linux package manager to download. You can ask AI

**Comment by sdzhengyang** [comment]
最后一个“查看文件里的字符串”，直接用010Editor或者WinHex打开，可以在文件末尾直接发现flag。不知道原理是不是跟strings工具相同。

**Reply by n-WN** [reply]
> 最后一个“查看文件里的字符串”，直接用010Editor或者WinHex打开，可以在文件末尾直接发现flag。不知道原理是不是跟strings工具相同。

是的
